lemur/tests/test_integration_db_and_models.py

```diff
@@ -0,0 +1,239 @@
+# This file consists of testing for the creation of objects and their
+# relationship in db
+import sys
+sys.path.append('../..')
```

Comment:
I really don't think that should be necessary, and part of what makes me think you should be investigating a test runner like nose.